source/features/conversation-activity-filter.css

```diff
@@ -1,20 +1,30 @@
 .rgh-conversation-activity-is-filtered .rgh-conversation-activity-filtered,
-.rgh-conversation-activity-is-collapsed-filtered .rgh-conversation-activity-collapsed,
+.rgh-conversation-activity-is-collapsed-filtered :is(
+	.rgh-conversation-activity-collapsed,
+	.js-resolvable-timeline-thread-container[data-resolved='true'], /* Resolved review thread */
+	.js-resolvable-timeline-thread-container[data-resolved='false'] .minimized-comment, /* Minimized review thread comment */
```

Comment:
Hides minimized review comments in unresolved threads:

![image](https://user-images.githubusercontent.com/46634000/162580740-fb9ead7e-0856-4fea-ab79-9ed4a16d34a9.png)